src/commands/new.ts

```diff
@@ -158,6 +158,13 @@ export interface Options {
    * @default mst
    */
   state?: StateMgmt
+  /**
+   * Whether or not to enable the New Architecture
+   *
+   * Input Source: `prompt.ask` | `parameter.option`
+   * @default false
+   */
+  newArch?: boolean
```

Comment:
Can you walk me through why you'd move this to its own flag rather than keeping it on the experimental side?

Is it because you want to keep the ability to drop back to old arch for a while?

Reply:
Since Expo/Meta is using the new arch by default, wouldn't it be better to have 
https://github.com/infinitered/ignite/blob/97ec737c6803f816474d67d6f72e88b90d351969/src/commands/new.ts#L165
and https://github.com/infinitered/ignite/blob/97ec737c6803f816474d67d6f72e88b90d351969/src/commands/new.ts#L521
as true instead, and keep the ability to drop back to old arch if needed?

Reply:
@Iannery to align with them directly, technically yes. but we're discussing internally here whether or not we are ready to make that jump as we're still converting many of our client projects over, which is uncovering issues with several libraries we must use

but yeah that's why this is still a draft, that default is up for discussion